src/wallet.rs

```diff
@@ -794,7 +756,7 @@ mod tests {
         let path = WalletPath::from(path);
 
         // we'll need a password too
-        let pwd = blake3::hash("mypassword".as_bytes());
+        let pwd = blake3::hash("mypassword".as_bytes()).as_bytes().to_vec();
```

Comment:
Can you open a follow up issue for this? https://github.com/dusk-network/wallet-cli/pull/171#discussion_r1280894623

Reply:
Added the follow up comment here https://github.com/dusk-network/wallet-cli/issues/162